src/v2/intents_client.js

```diff
@@ -784,6 +772,13 @@ class IntentsClient {
    *   The collection of intents to update or create.
    *
    *   This object should have the same structure as [IntentBatch]{@link google.cloud.dialogflow.v2.IntentBatch}
+   * @param {string} [request.languageCode]
```

Comment:
It looks like other changes snuck in here.  Can we keep these separate?

Reply:
yes, sure, I will separate these changes.